Content.Server/Traits/Assorted/AccentlessComponent.cs

```diff
@@ -6,5 +8,8 @@ namespace Content.Server.Traits.Assorted;
 [RegisterComponent]
 public sealed class AccentlessComponent : Component
 {
-
+    /// <summary>
+    ///     The accents removed by the accentless trait.
+    /// </summary>
+    public readonly List<Type> RemovedAccents = new() { typeof(ReplacementAccentComponent), typeof(LizardAccentComponent) };
```

Comment:
Use compregistry and datafield plz.

Reply:
What do you mean by datafield?

Reply:
I also don't know how I would use a component registry here.

Reply:
CompRegistry gives you the component types and is serializable and validates component existence.

Reply:
It has been switched over.